specification.markdown

```diff
@@ -365,29 +371,29 @@ The `type` attribute is OPTIONAL.
 <?xml version="1.0" encoding="utf-8"?>
 <EBMLSchema docType="files-in-ebml-demo" version="1">
  <!-- Root Element-->
- <element name="Files" level="0" id="0x1946696C" type="master">
+ <element name="Files" parent="/" level="0" id="0x1946696C" type="master">
   <documentation lang="en" type="definition">Container of data and
   attributes representing one or many files.</documentation>
-  <element name="File" level="1" id="0x6146" type="master" minOccurs="1"
+  <element name="File" parent="/Files" level="1" id="0x6146" type="master" minOccurs="1"
   maxOccurs="unbounded">
    <documentation lang="en" type="definition">An attached file.
    </documentation>
-   <element name="FileName" level="2" id="0x614E" type="utf-8"
+   <element name="FileName" parent="/Files/File" level="2" id="0x614E" type="utf-8"
    minOccurs="1">
     <documentation lang="en" type="definition">Filename of the attached
     file.</documentation>
    </element>
-   <element name="MimeType" level="2" id="0x464D" type="string"
```

Comment:
I prefer to simply say `parent="File"` with the Parent Element Name only and not the full path.